File: 백준/Bronze/2581. 소수/소수.cc
#include <iostream>
#include <vector>

bool isPrime(int n) {
  for (int i = 2; i < n; i++) {
    if (n % i == 0) return false;
  }
  return true;
}

int main() {
  int m;
  std::cin >> m;
  int n;
  std::cin >> n;
  std::vector<int> primes;
  int sum = 0;

  for (int i = m; i <= n; i++) {
    if (i != 1 && isPrime(i)) {
      primes.push_back(i);
      sum += i;
    }
  }

  if (primes.size() == 0) {
    std::cout << "-1";
  } else {
    std::cout << sum << std::endl << primes[0];
  }

  return 0;
}

File: 백준/Bronze/2581. 소수/소수.py
import sys

def is_prime(n: int) -> bool:
    if n < 2:
        return False
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True

def main() -> None:
    data = sys.stdin.read().strip().split()
    if len(data) < 2:
        return
    m, n = map(int, data)
    primes = [num for num in range(m, n + 1) if is_prime(num)]
    if not primes:
        print(-1)
        return
    print(sum(primes))
    print(primes[0])

if __name__ == '__main__':
    main()
